docs/source-2.0/guides/client-guidance/application-protocols/http.md

```diff
@@ -0,0 +1,179 @@
+# HTTP
+
+HTTP is the most common application protocol used by Smithy clients. This guide
+provides advice on how to integrate and expose HTTP clients.
+
+## Configuration
+
+Smithy clients should allow their users to configure the HTTP client that the
+Smithy client uses to send requests. Users may want to change the client used to
+something that has different performance characteristics or support for features
+that they need.
+
+## HTTP interfaces
+
+Smithy clients should provide interfaces for HTTP clients that standardize how
+the Smithy client interacts with the HTTP client, allowing any HTTP client to be
+used as long as it implements the interface.
+
+### Fields
+
+Most users who have interacted with HTTP directly are familiar with the concept
+of headers. Headers were originally introduced in HTTP/1.0 and, since then, the
+concept of key/value pairs has expanded to include trailers and other arbitrary
+metadata. As of {rfc}`9110`, these key/value pairs are exclusively referred to
+as {rfc}`fields <9110#section-5>`.
```

Comment:
I presume this `{rfc}` code is some kind of template used by Sphinx but obviously it doesn't render properly when [the file is viewed on GitHub](https://github.com/smithy-lang/smithy/blob/0abad2a4bc9c4d457e79a5f50072cfdb5cc08d43/docs/source-2.0/guides/client-guidance/application-protocols/http.md). How do we see the final rendered output?